cli/src/deploy/mod.rs

```diff
@@ -22,6 +23,9 @@ const FORBIDDEN_OPERATION: &str =
   "Please review your credentials. Make sure you have follow all the \
   configuration steps: https://alantech.gitbook.io/anycloud/";
 const NAME_CONFLICT: &str = "Another application with same app ID already exists.";
+const UNAUTHORIZED_OPERATION: &str =
+  "Invalid AnyCloud authentication credentials. Please try again and you will be asked \
+  to login once again.";
```

Comment:
A little awkward to have "again" twice in a sentence like that. Maybe "Please retry and you will be asked to reauthenticate." ?

Reply:
yes!